### PR TITLE
New package: waifu2x-converter-cpp-5.2

### PR DESCRIPTION
**Pull request by SolitudeSF**

### CODE REVIEW

**Comment by maxice8** [comment]
cross-compilation is failing, ,can't execute compiled file.

**Comment by SolitudeSF** [comment]
i have no clue whats the problem

**Comment by maxice8** [comment]
On Fri, Apr 06, 2018 at 08:51:00PM +0000, Solitude wrote:
> i have no clue whats the problem
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.*
> 

it compiles a binary and then tries to run it.

**Comment by SolitudeSF** [comment]
oh, i guess it generates arm executable

**Comment by SolitudeSF** [comment]
can i somehow prevent it from crosscompiling intermediate executable or should i just slap nocross on this?

**Comment by maxice8** [comment]
On Fri, Apr 06, 2018 at 01:59:15PM -0700, Solitude wrote:
> can i somehow prevent it from crosscompiling intermediate executable or should
> i just slap nocross on this?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.*
> 

Look at the CMakeLists.txt if there is an option() to point to a host binary, if not better slap nocross

**Comment by SolitudeSF** [comment]
uhh, im not sure. even if there is an option, how does it help if binary get compiled for target architecture and not the host one?

File: srcpkgs/waifu2x-converter-cpp/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+configure_args="-DINSTALL_MODELS=ON"
+hostmakedepends="pkg-config"
+makedepends="libopencv-devel opencl2-headers"
+short_desc="Image upscaling using deep convolutional neural networks"
+maintainer="SolitudeSF <solitudesf@protonmail.com>"
+license="MIT"
+homepage="https://github.com/DeadSix27/waifu2x-converter-cpp"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=a872f459e731bb324d13de801e0653385fa76260643f01a4fd1b24d66519063f
+nocross=yes
```
Comment (newbluemoon) [inline srcpkgs/waifu2x-converter-cpp/template]: Please add a reason why: `nocross="<reason/url to build log>"`

**Pull request closed without merging**